src/view/com/post-thread/PostThreadItem.tsx

```diff
@@ -402,7 +402,7 @@ let PostThreadItemLoaded = ({
                       type="lg"
                       style={pal.textLight}>
                       <Text type="xl-bold" style={pal.text}>
-                        {formatCount(post.repostCount)}
+                        {FormatCount(post.repostCount)}
```

Comment:
Why was this function capitalized?

Reply:
I resolve the linting error. It reverted to lowercase.